src/components/AccountBottomBar.tsx

```diff
@@ -145,67 +145,89 @@ const AccountState: React.FC<{
-    identifiers.length > 0 ? identifiers[0] : null
+    identifiers.length > 0 ? identifiers[0] : null,
   );
 
   const { x, y } = useMousePosition();
-  const [panelHeight, setPanelHeight] = useState(STORAGE_PANEL_MIN_HEIGHT);
-  const [isResizing, setIsResizing] = useState(false);
+  const [storageHeight, setStorageHeight] = useState(STORAGE_PANEL_MIN_HEIGHT);
+  const [resultHeight, setResultHeight] = useState(180);
+  const [isResizingStorage, setIsResizingStorage] = useState(false);
+  const [isResizingResult, setIsResizingResult] = useState(false);
 
-  const toggleResizing = (toggle: boolean) => {
-    setIsResizing(toggle);
+  const toggleResizingStorage = (toggle: boolean) => {
+    setIsResizingStorage(toggle);
+  };
+
+  const toggleResizingResult = (toggle: boolean) => {
+    setIsResizingResult(toggle);
   };
 
   const toggleResizeListener = () => {
-    toggleResizing(false);
+    toggleResizingStorage(false);
+    toggleResizingResult(false);
   };
 
   useEffect(() => {
     if (
-      isResizing &&
-      y > STORAGE_PANEL_MIN_HEIGHT &&
+      isResizingStorage &&
+      y > STORAGE_PANEL_MIN_HEIGHT - 30 + resultHeight &&
+      y < window.innerHeight - PLAYGROUND_HEADER_HEIGHT
+    ) {
+      setStorageHeight(y - resultHeight);
+    }
+  }, [x, y]);
+
+  useEffect(() => {
+    if (
+      isResizingResult &&
+      y > RESULT_PANEL_MIN_HEIGHT &&
       y < window.innerHeight - PLAYGROUND_HEADER_HEIGHT
     ) {
-      setPanelHeight(y);
+      setResultHeight(y);
     }
   }, [x, y]);
 
   useEffect(() => {
-    window.addEventListener("mouseup", toggleResizeListener, false);
+    window.addEventListener('mouseup', toggleResizeListener, false);
     return () => {
-      window.removeEventListener("mouseup", toggleResizeListener, false);
+      window.removeEventListener('mouseup', toggleResizeListener, false);
     };
   }, []);
 
   return (
-    <div>
-      <AccountStateContainer height={panelHeight}>
-        {renderDeployButton()}
+    <>
+      { identifiers.length ? <AccountStateContainer
+				height={storageHeight + resultHeight}
+      >
         <IdentifierList
           identifiers={identifiers}
           selected={selected}
-          onSelect={setSelected}
-          toggleResizing={toggleResizing}
+					onSelect={setSelected}
+					resize={() => toggleResizingStorage(true)}
+          controls={() => {
+            return (
+              <SidebarItemInsert grab={false}>
+								<></>
+              </SidebarItemInsert>
+            );
+          }}
         />
         <StateContainer value={storage[selected]} />
-      </AccountStateContainer>
-      <DeploymentResultContainer>
-        <Heading>
+      </AccountStateContainer> : ''}
+      <DeploymentResultContainer height={resultHeight}>
+        <ResizeHeading onMouseDown={() => toggleResizingResult(true)}>
           Deployment Result
           <ClearResults type={ResultType.Contract} />
-        </Heading>
+          <div></div>
```

Comment:
Do we need it here? 🤔
That's the reason why `Clear` button is not right aligned...